chalk-solve/src/solve/slg.rs

```diff
@@ -564,6 +567,38 @@ impl<I: Interner> MayInvalidate<'_, I> {
         true
     }
 
+    // Returns true if the two consts could be unequal.
+    fn aggregate_consts(&mut self, new: &Const<I>, current: &Const<I>) -> bool {
```

Comment:
This is duplicated in `aggregate`? (Or it's similar)

Reply:
It's similar